jetcd-launcher/src/main/java/io/etcd/jetcd/launcher/EtcdContainer.java

```diff
@@ -252,6 +253,12 @@ private void setDataDirectoryPermissions(String permissions) {
 
   private static Path createDataDirectory(String name) {
     try {
+      // https://github.com/etcd-io/jetcd/issues/489 - don't fail for Mac OS because of docker thing with /var/folders
+      final String os = System.getProperty("os.name").toLowerCase();
+      if (os.contains("mac")) {
+        final Path tmpDir = Paths.get("/private/var/tmp");
```

Comment:
I'm a little reluctant to add such hard coded path to the code, maybe we can try resolve the symlink to the real path or use a system property and a maven profile

Reply:
Sorry I'm not sure I understand what solution do you have in mind. Could you please elaborate on symlink (as far as I can tell `/var/folders/...` is not symlink) and system property?

Reply:
/var is a symlink to private/var

Reply:
Nice, resolving to real path solved the problem and it doesn't look like if is needed anymore. Thanks for suggestion!